bitrise/util.go

```diff
@@ -212,23 +210,6 @@ func SetBuildFailedEnv(failed bool) error {
 	return os.Setenv("BITRISE_BUILD_STATUS", statusStr)
 }
 
-// SaveConfigToFile ...
-func SaveConfigToFile(pth string, bitriseConf models.BitriseDataModel) error {
```

Comment:
Public method, but appears unused: https://github.com/search?q=org:bitrise-io+SaveConfigToFile+NOT+repo:bitrise-io/bitrise&type=code